src/main/java/subway/models/Line.java

```diff
@@ -0,0 +1,46 @@
+package subway.models;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Line {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(length = 20, nullable = false)
+    private String name;
+    @Column(length = 20, nullable = false)
+    private String color;
+    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<Section> sections = new ArrayList<>();
+
+    @Builder
+    private Line(String name, String color) {
+        this.name = name;
+        this.color = color;
+        this.sections = new ArrayList<>();
+    }
```

Comment:
```suggestion
    private Line(String name, String color, Station upStation, Station downStation) {
        this.name = name;
        this.color = color;
        this.sections = List.of(new Section(....);
    }
```

이런형태로 하면 구간없는 노선이 생성될 위험을 줄일 수 있을 것 같아요 😄